### ISSUE TITLE
NoPCM Case Issues

### ISSUE THREAD

**Issue by halonazhao** [JacquesCarette/literate-scientific-software#418]
Hello @szymczdm ,
I found a bunch of problems in this case I am not sure about. Please have a look. Thanks!

1.  For section 4.2.1 Assumptions, in manual version, there are abbreviations as references after those words, like theoretical model [T]. But in stable version, those words mapping to a specific section, like theoretical model [section 4.2.2]. Which is better? Which one I should keep?

2. For section 4.2.2 Theoretical Models, in manual version, the table includes Source and Ref. By information, but stable version don't. Should I keep them? Same questions for Section 4.2.4 DD, 4.2.5 IM, should I keep Source and Ref info?

3.  For Section 4.2.5 IM, in manual version there are input and output showing the equation, but in stable version, equation replaced input and output. Which one should I keep?

4. Small errors for IM2, in description, the last line. The manual version says the temperature 0 to 100, but stable says 0 to 0. I guess 0 to 0 is wrong?

5. Section 5.1 Function Requirements, in R2, the equations of mw are different  in Manual and Stable. Which is right?

**Comment by szymczdm** [comment]
@smiths can you double check I've mentioned everything that applies here and create any issues I missed?

1. Both should have the section references in the intro. The manual version has better references in each assumption (LC1, GD2, etc.).
2. Manual is correct, those fields should exist. However, we should be using citations instead of URLs (or have the option).
3. Manual is correct.
4. Manual is correct.
5. Manual is correct.

**Comment by smiths** [comment]
I agree with your summary @szymczdm.  I have also created a few issues of "to do" items that came out of this exercise.

**Comment by halonazhao** [comment]
All fixed. But still having minor problem with #424, I will leave comment under #424.

**Comment by smiths** [comment]
@halonazhao, where are the fixes committed?

**Comment by halonazhao** [comment]
I didn't upload. I thought I was just allowed to updating on my local machine. I will commit in another repo.

**Comment by smiths** [comment]
@halonazhao, you need to push to our case studies repo so that everyone can benefit from the fixes that you have made.

**Comment by halonazhao** [comment]
I have posted all the changes last term. I think all the issues I posted have been solved. Can I close this issue? @smiths
